### PR TITLE
feat: add `controller` to space filters

### PR DESCRIPTION
**Pull request by wa0x6e**
This PR add `controller` to `Space` filters, we we could get all spaces managed by a given address, like

```graphql
query Spaces {
  spaces(
    first: 20,
    skip: 0,
    orderBy: "created",
    orderDirection: desc,
    where: { controller: "0x24F15402C6Bb870554489b2fd2049A85d75B982f" }
  ) {
    id
  }
}
```

This avoid having the UI to lookup the address' domain, then send the space id with `id_in` when fetching all spaces managed by a given address (like in the "My Spaces" page)

This also enable API parity with SX api, which is already using the graphql API to show all spaces for a given controller, instead of resolving them UI side.

This is the first step of our transition into having a hardcoded `controller` property, on the space object

### TODO 

- [x] Merge https://github.com/snapshot-labs/stamp/pull/231

### CODE REVIEW

File: src/graphql/helpers.ts
Comment (ChaituVR) [inline src/graphql/helpers.ts:398]: Could we add a default stamp url if env doesn't exist?

File: src/graphql/helpers.ts
Comment (ChaituVR) [inline src/graphql/helpers.ts:405]: But we have other conditions to get space controller 🤔  Refer to this function https://github.com/snapshot-labs/snapshot.js/blob/master/src/utils.ts#L590-L607

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/graphql/helpers.ts
Comment (wa0x6e) [reply src/graphql/helpers.ts:405]: This is true, but this is not the method used by the UI to fetch all spaces controlled by an user.

See https://github.com/snapshot-labs/snapshot/blob/514f12c84d5986ef86f87193057d06d9b5c73353/src/views/ProfileAbout.vue#L59

The DNS check seems to be very hard to check live

**Review by ChaituVR — APPROVED** [review]
utAck

**Comment by ChaituVR** [comment]
Don't merge until we merge stamp

**Pull request merged**
